dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -254,20 +254,31 @@
 
       <!-- Hawkular Monitor agent subsystem -->
       <subsystem xmlns="urn:org.hawkular.agent.monitor:monitor:1.0"
-                 enabled="true"
                  apiJndiName="java:global/hawkular/agent/monitor/api"
                  numMetricSchedulerThreads="3"
                  numAvailSchedulerThreads="3">
+        <xsl:if test="$kettle.build.type='dev'">
```

Comment:
Would it make sense to use xsl:choose instead? See https://msdn.microsoft.com/en-us/library/ms256169(v=vs.110).aspx